### DISCUSSION TITLE
Manually transcribe audio, video, and image description

### DISCUSSION THREAD

**Discussion by rnoliveira** [sepinf-inc/IPED#1089] [Ideas]
Seria possível acrescentar uma nova aba ao lado da aba "Pré-visualização" com a listagem dos áudios, vídeos e imagens e uma caixa de texto logo abaixo para o operador inserir a degravação dos áudios/vídeos e descrever as imagens, presentes em extrações de dispositivos e nuvens, para as mesmas (as degravações e as descrições) fossem incorporadas ao chat quando da geração do relatório de evidências?

**Comment by rnoliveira** [comment]
Estou em uma equipe que utiliza bastante a ferramenta e realizamos a degravação de forma externa.

**Comment by lfcnassif** [comment]
Almost everything is possible to implement in software. Unfortunately I have a lot of other things scheduled to work on the next months. But this can be a (far) future improvement, or maybe someone of your team/organization could help to implement, I would be happy to review and test.

**Comment by rnoliveira** [comment]
Gostaria bastante de implementar essa mudança... Entretanto estamos na atividade fim... Teremos de esperar então, Vlw!
